libr/debug/p/debug_native.c

```diff
@@ -278,15 +278,19 @@ static RDebugInfo* r_debug_native_info (RDebug *dbg, const char *arg) {
 #if __WINDOWS__ && !__CYGWIN__
 static bool tracelib(RDebug *dbg, const char *mode, PLIB_ITEM item) {
 	const char *needle = NULL;
+	int tmp = 0;
 	if (mode) {
 		switch (mode[0]) {
 		case 'l': needle = dbg->glob_libs; break;
 		case 'u': needle = dbg->glob_unlibs; break;
 		}
 	}
-	eprintf ("(%d) %sing library at %p (%s) %s\n", item->pid, mode,
+	r_cons_printf ("(%d) %sing library at %p (%s) %s\n", item->pid, mode,
```

Comment:
i think i prefer eprintf here.. because thats an event, r_cons_ stuff will be only printed when the prompt is displayed.. which may be handy for r2pipe.. but i dont think this info should be handled this way